moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_objects.cpp

```diff
@@ -827,10 +840,12 @@ void MotionPlanningFrame::renameCollisionObject(QListWidgetItem* item)
     if (obj)
     {
       known_collision_objects_[item->type()].first = item_text;
+      // const Eigen::Isometry3d pose = obj->id_;
```

Comment:
delete

Reply:
Done/cleaned up the TODOs around those lines.